packages/api/src/commands/list.ts

```diff
@@ -0,0 +1,42 @@
+import chalk from 'chalk';
+import { Command } from 'commander';
+
+import logger from '../logger.js';
+import Storage from '../storage.js';
+
+const cmd = new Command();
+cmd
+  .name('list')
+  .alias('ls')
+  .description('list any installed API SDKs')
+  .action(async () => {
+    // We need to preload the storage system so we can grab the lockfile.
+    Storage.setStorageDir();
```

Comment:
This will end up creating the `.api/` directory, and an empty `api.json` file, if it doesn't exist which is probably fine? If you're using `npx api ls` to see if you have APIs installed then you're likely already pretty familiar with the library and having a sort of empty `.api/` directory might not be a big deal.